app/shared-components/PromoModal/PromoModal.jsx

```diff
@@ -76,7 +76,7 @@ class PromoModal extends React.Component {
 				break;
 			case INSIGHTS:
 				sendMessage('ping', 'promo_modals_insights_upgrade_cta');
-				url = `https://checkout.${DOMAIN}.com/insights?utm_source=gbe&utm_campaign=${utm_campaign}`;
+				url = `https://www.ghostery.com/insights/?utm_source=gbe&utm_campaign=${utm_campaign}`;
```

Comment:
looks like we do actually have a staging website now, rather than doubling dev work lets wait till support for this is done by @benstrumeyer in #595 to update this PR.